app/api/admin_statistics_api/events.py

```diff
@@ -50,3 +53,405 @@ class AdminStatisticsEventDetail(ResourceDetail):
         'class': NoModelLayer,
         'session': db.session
     }
+
+
+class AdminStatisticsEventTypeSchema(Schema):
```

Comment:
Like other APIs, this should be in the schema folder to make it readable and follow the standards.